keccak256/src/gates/iota_b9.rs

```diff
@@ -9,6 +9,7 @@ use std::marker::PhantomData;
 
 #[derive(Clone, Debug)]
 pub struct IotaB9Config<F> {
+    #[allow(dead_code)]
     q_enable: Selector,
```

Comment:
Some selectors are not enabled in the struct. But looks like they'll be fixed in #144 soon, so I use the `#[allow(dead_code)]` flag to make it pass here.

Reply:
Not a problem from my side with that :)